server/core/resources/firewall.bat

```diff
@@ -1,17 +1,49 @@
 @echo off
+:: Firewall modification doesn't work w/o admin, slimes will be sad.
+net session >nul 2>&1
+if %errorlevel% == 0 (
+    echo Running with administrative privileges! - Needed for firewall modification!
+) else (
+    echo Requesting administrative privileges - Needed for firewall modification! 
+    :: Temp script to request admin... Works and doesn't leave a mess.
+    echo Set UAC = CreateObject^("Shell.Application"^) > "%temp%\getadmin.vbs"
+    echo UAC.ShellExecute "%~s0", "", "", "runas", 1 >> "%temp%\getadmin.vbs"
+    "%temp%\getadmin.vbs"
+    exit /B
+)
+
 echo Installing firewall rules...
 
 rem Discovery default port
-netsh advfirewall firewall add rule name="SlimeVR UDP 35903 incoming" dir=in action=allow protocol=UDP localport=35903
-netsh advfirewall firewall add rule name="SlimeVR UDP 35903 outgoing" dir=out action=allow protocol=UDP localport=35903
-
+call :AddRule "SlimeVR UDP 35903 incoming" "dir=in action=allow protocol=UDP localport=35903 enable=yes"
+call :AddRule "SlimeVR UDP 35903 outgoing" "dir=out action=allow protocol=UDP localport=35903 enable=yes"
```

Comment:
```suggestion
```
I only see this port number used here in the code:
https://github.com/SlimeVR/SlimeVR-Server/blob/e2a511b552d52ebc77da0cb029440600edd5ddfb/server/desktop/src/main/java/dev/slimevr/desktop/Main.kt#L99

In a close statement, so don't think it is used anywhere

Reply:
To be fair, I kept all the rules already existing in the old script.

Reply:
this can be removed, @Eirenliel do you know where this comes from?

Reply:
I say we get rid of those rules, there's no need anymore